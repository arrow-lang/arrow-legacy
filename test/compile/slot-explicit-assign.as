let value: int8;

value = 3;
