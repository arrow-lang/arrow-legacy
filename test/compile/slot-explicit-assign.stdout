@value = internal global i8 0

define internal void @"slot-explicit-assign.@init"() {
  store i8 3, i8* @value
  ret void
}

define i32 @main(i32, i8**, i8**) {
  call void @"slot-explicit-assign.@init"()
  ret i32 0
}
